packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -42,6 +42,20 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 			await this.importFile(filePaths[i], parentFolderId);
 		}
 
+		try {
+			for (const importedLocalPath of Object.keys(this.importedNotes)) {
+				const note = this.importedNotes[importedLocalPath];
+				const updatedBody = await this.importLocalFiles(importedLocalPath, note.body, note.parent_id);
+				const updatedNote = {
+					...this.importedNotes[importedLocalPath],
+					body: updatedBody || note.body,
+				};
+				this.importedNotes[importedLocalPath] = await Note.save(updatedNote, { isNew: false });
+			}
+		} catch (error) {
+			// console.error(`Problem importing links for file ${resolvedPath}, error:\n ${error}`);
```

Comment:
That would need to be addressed - and in fact shouldn't we actually throw an error here?

Reply:
Maybe just removing the try catch clause is the best solution. I end up just copying the original implementation.